### PR TITLE
Docs-3487 SCALE 22.02 Reorg and Update Reporting Article Content

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit copies CORE reporting content into SCALE and updates it to SCALE specific where applicable. It creates a Reporting section in Tutorials for ConfigReportsSCALE.md and the UI ref article ReportingScreensSCALE.md. It updates the snippet used in both CORE and SCALE.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Reporting/ConfigReportsScale.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+{{< /hint >}}
+
+Data files are saved in <file>/var/db/collectd/rrd/</file>.
+
+## Configuring Report Settings
+Click the <i class="material-icons" aria-hidden="true" title="Settings">settings</i> to open the **Reports Configuration** configuration screen where you control how TrueNAS displays the graphs.
+
+![ReportsConfigurationSCALE](/images/SCALE/22.02/ReportsConfigurationSCALE.png "Reports Configuration")
+
+Select the general options you want to used in your TrueNAS.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/Reporting/ConfigReportsScale.md:38]:
```suggestion
Select the general options you want to use in your TrueNAS.
```

File: content/SCALE/SCALETutorials/Reporting/ConfigReportsScale.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Specify either the host name or IP address of the [Graphite](https://graphiteapp.org/) server you want to use.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/Reporting/ConfigReportsScale.md:40]:
```suggestion
Specify either the hostname or IP address of the [Graphite](https://graphiteapp.org/) server you want to use.
```

File: content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+alias: /scale/scaleuireference/reporting/
+tags:
+ - scalereports
+---
+
+
+{{< toc >}}
+
+
+The **Reporting** screen displays graphs of system information for CPU, disk, memory, network, NFS, partition, target, UPS, ZFS and system functions. The **CPU** report displays by default.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md:14]:
```suggestion
The **Reporting** screen displays graphs of system information for CPU, disk, memory, network, NFS, partition, target, UPS, ZFS, and system functions. The **CPU** report displays by default.
```

File: content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+The <i class="material-icons" aria-hidden="true" title="Settings">settings</i> opens the **[Reports Configuration](#reports-configuration-screen)** configuration screen.
+
+## Reports Configuration Screen
+**Reports Configuration** settings specify how TrueNAS displays the graphs and the host name of the Graphite server.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md:19]:
```suggestion
**Reports Configuration** settings specify how TrueNAS displays the graphs and the hostname of the Graphite server.
```

File: content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+| **Network** | Displays an **Interface Traffic** graph for each interface in the system. |
+| **NFS** | Displays the **NFS Stats (Operations)** and **NFS Stats (Bytes)** graphs. |
+| **Partition** | Displays graphs showing disk space allocations.  |
+| **System** | Displays both the **Processes** and **Uptime** graphs. |
+| **Target** | Displays graphs only for systems with iSCSI ports configured and shows the bandwidth statistics for iSCSI ports. |
+| **UPS** | Displays the graphs only if the system is configured for and uses a UPS. |
+| **ZFS**| Displays the **ARC Size**, **ARC Hit Ratio**, **ARC Requests demand_data**, **ARC Requests demand_metadata**, **ARC Requests prefetch_data**, and **ARC Requests prefetch_metadata** graphs with the Arc and L2 gigabytes and hits (%), and the hits, misses and total number of requests. |
+
+## Report Graphs
+The following sections provide example of each report graph.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md:41]:
```suggestion
The following sections provide examples of each report graph.
```

File: content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+### CPU Graphs
+{{< expand "Click Here for More Information" "v" >}}
+[CPU](https://collectd.org/wiki/index.php/Plugin:CPU) graphs shows the amount of time spent by the CPU in various states such as executing user code, executing system code, and being idle.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md:44]:
```suggestion
[CPU](https://collectd.org/wiki/index.php/Plugin:CPU) graphs show the amount of time spent by the CPU in various states such as executing user code, executing system code, and being idle.
```

File: content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+{{< expand "Click Here for More Information" "v" >}}
+[NFS](https://collectd.org/wiki/index.php/Plugin:NFS) graphs show information about the number of calls for each procedure and whether the system is a server or client.
+
+![ReportingNFSGraphsSCALE](/images/SCALE/22.02/ReportingNFSGraphsSCALE.png "NFS Reporting")
+
+{{< /expand >}}
+### Partition Graphs
+{{< expand "Click Here for More Information" "v" >}}
+
+[Partition](https://collectd.org/wiki/index.php/Plugin:DF) graphs displays free, used, and reserved space for each pool and dataset. However, the disk space used by an individual zvol is not displayed as it is a block device.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md:94]:
```suggestion
[Partition](https://collectd.org/wiki/index.php/Plugin:DF) graphs display free, used, and reserved space for each pool and dataset. However, the disk space used by an individual zvol is not displayed as it is a block device.
```

File: content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+
+![ReportingPartitionGraphsSCALE](/images/SCALE/22.02/ReportingPartitionGraphsSCALE.png "Partition Reporting")
+
+{{< /expand >}}
+### System Graphs
+{{< expand "Click Here for More Information" "v" >}}
+[System](https://collectd.org/wiki/index.php/Plugin:Processes) graphs displays the number of processes grouped by state, sleeping, running, stopped, zombies and blocked, and system uptime.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md:101]:
```suggestion
[System](https://collectd.org/wiki/index.php/Plugin:Processes) graphs display the number of processes grouped by state, sleeping, running, stopped, zombies and blocked, and system uptime.
```

File: content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+### UPS Graphs
+{{< expand "Click Here for More Information" "v" >}}
+[UPS](https://collectd.org/wiki/index.php/Plugin:NUT)graphs show statistics about an uninterruptible power supply (UPS) using [Network UPS tools](https://networkupstools.org/). Statistics include voltages, currents, power, frequencies, load, and temperatures. 
+
+![ReportingUPSTab](/images/CORE/12.0/ReportingUPSTab.png "UPS Reporting")
+
+{{< /expand >}}
+### ZFS Graphs
+{{< expand "Click Here for More Information" "v" >}}
+[ZFS](https://collectd.org/wiki/index.php/Plugin:ZFS_ARC) graphs shows compressed physical ARC size, hit ratio, demand data, demand metadata, and prefetch data and metadata.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md:121]:
```suggestion
[ZFS](https://collectd.org/wiki/index.php/Plugin:ZFS_ARC) graphs show compressed physical ARC size, hit ratio, demand data, demand metadata, and prefetch data and metadata.
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]

File: content/SCALE/SCALETutorials/Reporting/ConfigReportsScale.md (hunk 2 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Reporting/ConfigReportsScale.md:40]: host name is two words unless you are talking about the CLI command hostname.

File: content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md (hunk 4 above)
Comment (micjohnson777) [reply content/SCALE/SCALEUIReference/Reporting/ReportingScreensSCALE.md:19]: host name in this instance is two words because it is not the CLI command

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**